### DISCUSSION TITLE
Question: how to add a summary node?

### DISCUSSION THREAD

**Discussion by fwfsf** [freeplane/freeplane#1615] [Help]
How to add a summary node to e and f?

![question](https://github.com/freeplane/freeplane/assets/107751295/29e18e3a-254c-4373-abb6-4eba666a6cc2)

**Comment by dpolivaev** [comment]
Add it to b and d

**Comment by fwfsf** [comment]
Using b and d still fails.

![bd](https://github.com/freeplane/freeplane/assets/107751295/ade1161c-fd48-4947-900f-2d2c23bb22e4)

I want to implement the following structure:

![gg](https://github.com/freeplane/freeplane/assets/107751295/d4337c21-80bb-4c63-9054-a821a4ad283b)

Is there any way to implement it?

**Comment by dpolivaev** [comment]
I checked it and yes it is not supported: 

Node f is a first-level summary node.
A summary node containing f should be a second-level summary node.
But neither node b nor node e is a summary node.

Therefore, you can not have a summary node that summarizes b or e together with f.

**Comment by dpolivaev** [comment]
If e were a summary node of b, g could be summary node of e and f

<img width="354" alt="summarynodes" src="https://github.com/freeplane/freeplane/assets/1558052/45fddecc-e785-45c1-a255-a5b76e5e24e9">

**Comment by fwfsf** [comment]
I didn't notice that a single node could be turned into a summary node, which is a little weird. I think it would be better if this structure could be implemented without turning a single node into a summary node.

Thanks.

**Comment by EdoFro** [comment]
![image](https://github.com/freeplane/freeplane/assets/35700575/983c7767-c654-4604-9ee6-4ea99b8641c5)

**Reply by EdoFro** [reply]
![image](https://github.com/freeplane/freeplane/assets/35700575/4eb7fb96-69cd-42ba-ab7d-d3da9c43e579)

to hide a node:
- select it
- click menu command `View->Hidden node`
